static/locales/en-US.yaml

```diff
@@ -714,6 +714,8 @@ External link opening has been disabled in the general settings: 'External link
 Downloading has completed: 'Downloading "$" has completed'
 Starting download: 'Downloading "$" has started'
 Downloading failed: 'Unable to download "$", return http request status code $'
+Downloading canceled: The dowload is canceled by the user
+Download folder does not exist: The download repertory "$" doesn't exist. Falling back to "ask folder" mode.
```

Comment:
I think this one has a misspelling, "**repertory**" which I believe you wanted to give it as "**repository**" but for this position "**directory**" seems to be the correct one here.